src/components/WalletSelector/WalletButton.tsx

```diff
@@ -56,12 +59,22 @@ function WalletButton({
       return null;
     }
 
+    function convertWalletName(name: string) {
+      const overrides: Record<string, string> = {
```

Comment:
Maybe this `overrides` map can be defined outside of the component

Reply:
was thinking about this, but i think it's kinda nice to have it right where the logic is used (fewer mental hops)

in the future if we need the map in more places we can move